module/apps/actor-armor.js

```diff
@@ -26,10 +26,12 @@ export default class ActorArmorConfig extends DocumentSheet {
 
   /** @inheritdoc */
   async getData() {
-
     // Get actor AC data
-    const actorData = this.object.toObject(false).system;
-    const ac = actorData.attributes.ac;
+    const ac = this.object.constructor.computeArmorClass(
```

Comment:
I propose an alternative approach here. Instead of making `computeArmorClass` a method that can accept alternative inputs, I think the workflow for the `ActorArmorConfig` sheet should be to `clone()` the actor, apply changes to the clone via `updateSource()`, and then when the form is saved commit those changes back to the original actor.

This avoids us needing to have methods like `Actor#computeArmorClass` which operate on data other than the actor's own data which I think is preferable.